interfaces/builtin/system_observe.go

```diff
@@ -51,6 +51,7 @@ deny ptrace (trace),
 @{PROC}/vmstat r,
 @{PROC}/diskstats r,
 @{PROC}/kallsyms r,
+@{PROC}/version_signature r,
```

Comment:
Can you remove this and put in template.go as described in my previous comment a moment ago? This is a common access and we don't want applications to require the privileged system-observe for it.